packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/About.1.doc.mdx

```diff
@@ -30,17 +98,38 @@ This component requires the following peer dependencies be installed in your app
 | react-dom | ^16.8.5 |
 | react-intl | ^2.8.0 |
 
-<!-- AUTO-GENERATED-CONTENT:END -->
+## FolderTree props
```

Comment:
Updated to "Props Table" here: [3229bd1](https://github.com/cerner/terra-framework/pull/1973/commits/3229bd15cb1231e0372bee951487c4573c55f676)

`FolderTree` and `FolderTreeItem` seem more correct to me since these refer to the specific React components and not just "Folder Tree" or "Folder Tree Item" in general?